src/cockpit/bridge.py

```diff
@@ -21,18 +21,39 @@
 import logging
 import pwd
 import os
+import shlex
 import sys
 
 from systemd_ctypes import EventLoopPolicy
 
+from .channel import Channel
+from .channels import CHANNEL_TYPES
 from .transports import StdioTransport
 from .packages import Packages
 from .router import Router
 
-logger = logging.getLogger('cockpit.bridge')
+logger = logging.getLogger(__name__)
 
 
-async def run():
+class Bridge(Router):
+    def __init__(self, args: argparse.Namespace):
+        self.packages = Packages()
+        self.args = args
+
+        super().__init__(Channel.create_routing_rules(CHANNEL_TYPES))
+
+    def do_send_init(self) -> None:
+        with open('/usr/lib/os-release', encoding='utf-8') as file:
```

Comment:
Side note: I know I recently moved some `/etc/os-release` over to this, but turns out this was wrong: os-release(5) explicitly wants applications to check the one in /etc.

Reply:
I just re-read the manpage and it seems to suggest that we ought to check in both places, with precedence given to `/etc`.  The symlink is "to provide compatibility with applications only looking at /etc/."